src/msg.rs

```diff
@@ -0,0 +1,4 @@
+#![allow(unused_imports)]
+#![allow(dead_code)]
+use flatbuffers;
+include!(concat!(env!("OUT_DIR"), "/gen/msg_generated.rs"));
```

Comment:
This was always giving me trouble, because the generated rs file contained `#![allow]` because of that rust wouldn't let me import it. Did something change so it works now?

Reply:
Yes - https://github.com/google/flatbuffers/commit/636b516492594462be9cd3d0224dbdc27a2a423b